exercises/clock/clock.spec.js

```diff
@@ -70,140 +70,150 @@ describe('Clock', () => {
+    });
 
-      xtest('clocks with negative hour', () => {
-        expect(new Clock(22, 40).equals(new Clock(-2, 40))).toBeTruthy();
-      });
+    xtest('clocks with negative hour that wraps', () => {
+      expect(new Clock(17, 3).equals(new Clock(-31, 3))).toBeTruthy();
+    });
 
-      xtest('clocks with negative hour that wraps', () => {
-        expect(new Clock(17, 3).equals(new Clock(-31, 3))).toBeTruthy();
-      });
+    xtest('clocks with negative hour that wraps multiple times', () => {
+      expect(new Clock(13, 49).equals(new Clock(-83, 49))).toBeTruthy();
+    });
 
-      xtest('clocks with negative hour that wraps multiple times', () => {
-        expect(new Clock(13, 49).equals(new Clock(-83, 49))).toBeTruthy();
-      });
+    xtest('clocks with minute overflow', () => {
+      expect(new Clock(0, 1).equals(new Clock(0, 1441))).toBeTruthy();
+    });
 
-      xtest('clocks with minute overflow', () => {
-        expect(new Clock(0, 1).equals(new Clock(0, 1441))).toBeTruthy();
-      });
+    xtest('clocks with minute overflow by several days', () => {
+      expect(new Clock(2, 2).equals(new Clock(2, 4322))).toBeTruthy();
+    });
 
-      xtest('clocks with minute overflow by several days', () => {
-        expect(new Clock(2, 2).equals(new Clock(2, 4322))).toBeTruthy();
-      });
+    xtest('clocks with negative minute', () => {
+      expect(new Clock(2, 40).equals(new Clock(3, -20))).toBeTruthy();
+    });
 
-      xtest('clocks with negative minute', () => {
-        expect(new Clock(2, 40).equals(new Clock(3, -20))).toBeTruthy();
-      });
+    xtest('clocks with negative minute that wraps', () => {
+      expect(new Clock(4, 10).equals(new Clock(5, -1490))).toBeTruthy();
+    });
 
-      xtest('clocks with negative minute that wraps', () => {
-        expect(new Clock(4, 10).equals(new Clock(5, -1490))).toBeTruthy();
-      });
+    xtest('clocks with negative minute that wraps multiple times', () => {
+      expect(new Clock(6, 15).equals(new Clock(6, -4305))).toBeTruthy();
+    });
 
-      xtest('clocks with negative minute that wraps multiple times', () => {
-        expect(new Clock(6, 15).equals(new Clock(6, -4305))).toBeTruthy();
-      });
+    xtest('clocks with negative hours and minutes', () => {
+      expect(new Clock(7, 32).equals(new Clock(-12, -268))).toBeTruthy();
+    });
 
-      xtest('clocks with negative hours and minutes', () => {
-        expect(new Clock(7, 32).equals(new Clock(-12, -268))).toBeTruthy();
-      });
+    xtest('clocks with negative hours and minutes that wrap', () => {
+      expect(new Clock(18, 7).equals(new Clock(-54, -11513))).toBeTruthy();
+    });
 
-      xtest('clocks with negative hours and minutes that wrap', () => {
-        expect(new Clock(18, 7).equals(new Clock(-54, -11513))).toBeTruthy();
-      });
+    xtest('full clock and zeroed clock', () => {
+      expect(new Clock(24, 0).equals(new Clock(0, 0))).toBeTruthy();
```

Comment:
`toBeTruthy()` uses `!!` to check for `truthy` values. Which means things like non-empty strings, arrays would pass the test. We would like to check for the value to be exactly equal to `true`. Can you please change **all** `toBeTruthy()` to `toBe(true)` & `toBeFalsy()` to `toBe(false)`?

Ref: https://stackoverflow.com/a/32767435/3507206